src/ubuntu/18.04/mono/amd64/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-amd64
```

Comment:
I think we shouldn't derive from the Helix image since there's a lot of things we don't need from there in mono/mono, let's just copy whatever we need from there and inline here.